src/core/device_model.jl

```diff
@@ -82,9 +85,20 @@ function _set_model!(
     dict[key] = model
 end
 
-function _initialize_timeseries_labels(
+function initialize_timeseries_labels(
     ::Type{D},
     ::Type{T},
 ) where {D <: PSY.Device, T <: AbstractDeviceFormulation}
     return Dict{Type{<:TimeSeriesParameter}, String}()
 end
+
+function initialize_attributes(
+    ::Type{D},
+    ::Type{T},
+) where {D <: PSY.Device, T <: AbstractDeviceFormulation}
+    return Dict{String, Any}()
+end
+
+function apply_reservations(model::DeviceModel)
```

Comment:
We should use the get attribute function so that it is easier to read in the construction code. Since apply reservation isn't defined for all device models, this could be confusing

Reply:
ok, I'll remove this and use the getter function